Watchman.Engine/Sns/SnsTopicCreator.cs

```diff
@@ -19,20 +19,16 @@ public class SnsTopicCreator : ISnsTopicCreator
         public async Task<string> EnsureSnsTopic(string alertingGroupName, bool dryRun)
         {
             var topicName = alertingGroupName + "-Alerts";
-            var topic = await _snsClient.FindTopicAsync(topicName);
```

Comment:
`FindTopicAsync` is implemented client-side in the AWS SDK and involves fetching every single topic